packages/sales-api-service/src/schema/authenticate.schema.js

```diff
@@ -5,11 +5,10 @@ import { permitSchema } from './permit.schema.js'
 import { contactResponseSchema } from './contact.schema.js'
 import { finalisedPermissionSchemaContent } from './permission.schema.js'
 
+const referenceLength = 6
```

Comment:
By convention, we use `ALL_CAPS_WITH_UNDERSCORES` for constant values like this, so change to `REFERENCE_LENGTH`, please